internal/resolution/variable_sources/entity/bundle_entity.go

```diff
@@ -120,17 +118,10 @@ func (b *BundleEntity) ChannelProperties() (*ChannelProperties, error) {
 }
 
 func (b *BundleEntity) BundlePath() (string, error) {
-	if bundlePath, ok := b.Entity.Properties[propertyBundlePath]; ok && bundlePath != "" {
-		return bundlePath, nil
-	}
-	return "", fmt.Errorf("error determining bundle path for entity '%s': property '%s' not found or empty", b.ID, propertyBundlePath)
-}
-
-func (b *BundleEntity) DefaultChannel() (string, error) {
-	if defaultChannel, ok := b.Entity.Properties[propertyDefaultChannel]; ok && defaultChannel != "" {
-		return defaultChannel, nil
+	if err := b.loadBundlePath(); err != nil {
+		return "", err
 	}
-	return "", fmt.Errorf("error determining default channel for entity '%s': property '%s' not found or empty", b.ID, propertyDefaultChannel)
```

Comment:
If this was a useful error before, why did we lose it?

Reply:
Oops. Looks like this was part of the v0 CatalogSource integration PR (that got reverted as part of the first commit of this PR which reverts the entire thing). @ankitathomas @perdasilva if you could help me with cherry-picking the relevant parts of that PR that should be included back, that'll save me some time from sifting through the v0 integration PR 😀

Reply:
Default channel isn't a concept in OLMv1 yet anyway (and it may never be). So perhaps we just fully remove anything referencing default channels?